run/cifs_archive/archive-clips.sh

```diff
@@ -28,57 +28,58 @@ function connectionmonitor {
 
 function moveclips() {
   ROOT="$1"
-  PATTERN="$2"
+  SUB="$2"
```

Comment:
You could keep this a 2-argument function and just add `SUB=$(dirname "$ROOT")` here.
That way a lot of the code below can stay the same.

Reply:
(actually `SUB=$(basename $(dirname $ROOT))`)

Reply:
got it, makes it much simpler, thanks.